content/SCALE/SCALEUIReference/Apps/AppsScreensSCALE.md

```diff
@@ -0,0 +1,264 @@
+| **Containers** | Required. Select the container from the dropdown list. |
+| **Commands** | Enter the shell command. |
+
+Select **Choose** to open the **Applications > Pod Shell** screen. 
+{{< /expand >}}
+
+### Pod Shell Screen
+The **Pod Shell** screen allows users to enter TrueNAS CLI commands to access information about their applications.
+{{< expand "Click Here for More Information" "v" >}}
+
+![AppsPodShellWindow](/images/SCALE/22.02/AppsPodShellWindow.png "Applications Pod Shell")
+
+The following are examples of commands you can enter to access information on an active container. You can also use the **System Settings > Shell** to access the same information.
+
+To view container namespaces: `k3s kubectl get namespaces`.
+To view pods by namespace: `k3s kubectl get -n <NAMESPACE> pods`.
+To access container shell: `k3s kubectl exec -n <NAMESPACE> --stdin --tty <POD> -- /bin/bash`.
+To view details about all containers: `k3s kubectl get pods,svc,daemonsets,deployments,statefulset,sc,pvc,ns,job --all-namespaces -o wide`.
+To get container status: `k3s kubectl describe -n <CONTAINER NAMESPACE> <POD-ID>`.
+{{< /expand >}}
+
+### Choose Log Window
+The **Logs** options opens the **Choose Log** window.
+{{< expand "Click Here for More Information" "v" >}}
+
+![AppsChooseLogWindow](/images/SCALE/22.02/AppsChooseLogWindow.png "Choose Log Window") 
+
+| Setting | Description |
+|---------|-------------|
+| **Pods** | Select the pod from the dropdown list to open the shell screen with the log for this pod. |
+| **Containers** | Select the containers from the dropdown list to include in the log shell screen. |
+| **Tail Lines** | Enter the number of log file lines to include in the shell screen for the log file. |
+{{< /expand >}}
+
+### Pod Log Window
+The **Pod Log** shell screen displays with the information selected in the **Choose Log** window. 
+{{< expand "Click Here for More Information" "v" >}}
+
+![ApplicationsPodLogsScreen](/images/SCALE/22.02/ApplicationsPodLogsScreen.png "Applications Pod Logs Shell") 
+
+Use the **Set font size** slider to increase or decrease the size of the font displayed on the screen.
+**Reconnect** re-establishes a connection with the application service.
+**Download Logs** downloads the logs to your server.
+{{< /expand >}}
+
+## Available Applications
+The **Available Applications** screen displays the widgets for all applications in the **Official** catalog.
+{{< expand "Click Here for More Information" "v" >}}
+
+![AvailableApplicationsScreen](/images/SCALE/22.02/AvailableApplicationsScreen.png "Available Applications")
+
+The **Install** button on each application card opens the configuration wizard for that application.
+
+Click on the application icon or name to open an ***appname* Application Summary** window that includes information on the **Catalog**, **Categories**, **Train**, **Status** and **Versions** for that application.
```

Comment:
Line 205: is ***appname* formatted correctly?

Reply:
yes, in this instance appname is a variable so italic is correct.